docs/modules/development/pages/docs/build-docs.adoc

```diff
@@ -0,0 +1,82 @@
+:imagesdir: ../assets/images
+= Build documentation
+
+This section describes how to build documentation locally. 
+We recommend that you do this to test major documentation changes, particularly if adding a lot of new content to the documentation or making significant revisions to existing documentation. 
+
+== Install Antora
+
+Antora is a documenation site generator based around the AsciiDoc format. 
+To install Antora, follow the https://docs.antora.org/antora/2.2/install/install-antora/[instructions] on the Antora website.  
+
+== Make a copy of the OpenNMS docs repository
+
+If you already know how to use https://git-scm.com/[git], then this is going to sound very familiar.
+If not, here’s the full walk-through:
+
+. Install git.
+. Locate the OpenNMS repo you want to write docs for and https://help.github.com/articles/fork-a-repo/[fork] it. (For example, the https://github.com/OpenNMS/opennms-helm[OpenNMS Helm repo].)
```

Comment:
Should we use the OpenNMS repo here as an example for the lazy ones :)

Reply:
LOL. Ya, sure.